WS2012R2/lisa/remote-scripts/ica/provisionLinuxForLisa.sh

```diff
@@ -514,8 +529,11 @@ cat ~/${CONSTANTS_FILE}
 #
 distro=`LinuxRelease`
 case $distro in
-    "CENTOS" | "RHEL")
-	    RhelTasks
+    "CENTOS6" | "RHEL6")
+	    RhelTasks 6
```

Comment:
function RhelTasks is not updated to take parameters, it might just work as it is but it might not be the proper way to handle the release version tag.